frontend/src/screens/ProductScreen.tsx

```diff
@@ -18,32 +16,23 @@ type ProductScreenProps = {
   }
 }
 
-type ProductDetailsState = {
-  productDetails: {
-    loading: boolean
-    error: string
-    product: {
-      _id: string
-      name: string
-      image: string
-      description: string
-      price: number
-    }
-  }
-}
-
 export const ProductScreen = ({ history, match }: ProductScreenProps) => {
-  const qty = 1
-  const dispatch: Dispatch<any> = useDispatch()
+  const _id = { _id: match.params.id }
 
-  const productDetails = useSelector(
-    (state: ProductDetailsState) => state.productDetails
-  )
-  const { loading, error, product } = productDetails
+  console.log(_id)
```

Comment:
We can remove the console.log calls right?

Reply:
MB, removed